tox.ini

```diff
@@ -4,11 +4,115 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = flake8, py27, py33, py34, py35
+envlist =
+    py27,
+    py34,
+    py35,
+    py36,
+    docs,
+    isort,
+    flake8
+    coverage-report,
 
 [testenv]
-commands = python setup.py test
+usedevelop = True
+extras =
+    test
+
+commands =
+    python setup.py test
+    mkdir -p {toxinidir}/reports/coverage {toxinidir}/reports/pytest
+    pytest --cov=lib --cov-report=xml --html={toxinidir}/reports/pytest/report-{envname}.html --self-contained-html lib/diazo/tests {posargs}
+
+setenv =
+    COVERAGE_FILE=.coverage.{envname}
+
+deps =
+    pdbpp
+    pytest
+    pytest-cov
+    pytest-remove-stale-bytecode
+    pytest-mock
+    pytest-html
```

Comment:
@gforcada yes that would be possible, but I don't like that actually, cause I want readable configurations.

perfect would be something like:

    deps = 
        -c https://jenkins.plone.org/test-resources/constrains.txt
        pytest
        ...

and that should actually work, as tox just pass all of the dependency parameters to pip